recipes/actinia-api/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "actinia-api" %}
```

Comment:
Did you mean the `actinia-python-client`?
```suggestion
{% set name = "actinia-python-client" %}
```

Reply:
Thanks for the correction. I will close this PR and submit a new one.